mllm/backends/cpu/kernels/common/fa2_1/impl-any-simd-inst.hpp

```
@@ -0,0 +1,29 @@
+
+// Platform-specific definitions used for declaring an interface, independent of
+// the SIMD instruction set.
+#include <hwy/base.h>  // HWY_RESTRICT
+
+namespace mllm::cpu::flash_attn2::details {
+
+HWY_DLLEXPORT void call_vector_dot_product_fp32_fp32_fp32(const mllm_fp32_t* HWY_RESTRICT lhs,
+                                                          const mllm_fp32_t* HWY_RESTRICT rhs, mllm_fp32_t* HWY_RESTRICT out,
+                                                          size_t len);
+
+HWY_DLLEXPORT void call_fma_const_array_fp32(mllm_fp32_t* HWY_RESTRICT acc_o, mllm_fp32_t acc_s,
```

Clang (14.0.6)

[error] variable 'call_fma_const_array_fp32' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables,-warnings-as-errors)